plugins/plugin-site/src/components/PluginPageLayout.jsx

```diff
@@ -35,6 +35,7 @@ function PluginPageLayout({plugin, children}) {
         {id: 'releases', to: `/${plugin.name}/releases/`, label: 'Releases'},
         {id: 'issues', to: `/${plugin.name}/issues/`, label: 'Issues'},
         {id: 'dependencies', to: `/${plugin.name}/dependencies/`, label: 'Dependencies'},
+        {id: 'health-score', to: `/${plugin.name}/health-score/`, label: 'Health Score'},
```

Comment:
Kinda nitpicky, but lets do healthscore not health-score for a bit of consistency.

Reply:
done in https://github.com/jenkins-infra/plugin-site/pull/1431/commits/f1280dad2e0423dbf7fd3b9109998cdd16888b2b